### PR TITLE
Fix golint issues on the windows graph driver.

### PR DESCRIPTION
**Pull request by calavera**
Signed-off-by: David Calavera david.calavera@gmail.com

### CODE REVIEW

File: daemon/graphdriver/windows/windows.go [hunk 1]
```diff
@@ -26,23 +26,29 @@ import (
 	"github.com/microsoft/hcsshim"
 )
 
+// init registers the windows graph drivers to the register
```
Comment (runcom) [inline daemon/graphdriver/windows/windows.go]: is it really complaining for unexported fields as well?!

**Comment by jhowardmsft** [comment]
@swernli - can you glance over this please? :)

**Comment by runcom** [comment]
There's one missing I think:
```
daemon/graphdriver/windows/windows.go:259:1: exported method Driver.Cleanup should have comment or be unexported
```

File: daemon/graphdriver/windows/windows.go (hunk 1 above)
Comment (calavera) [inline daemon/graphdriver/windows/windows.go]: no, but we've also been documented unexported fields/functions.

**Comment by calavera** [comment]
@runcom fixed

**Comment by runcom** [comment]
LGTM once janky is happy

File: daemon/graphdriver/windows/windows.go [hunk 2]
```diff
@@ -26,23 +26,29 @@ import (
...
 func init() {
 	graphdriver.Register("windowsfilter", InitFilter)
 	graphdriver.Register("windowsdiff", InitDiff)
 }
 
 const (
+	// diffDriver is a constant to
```
Comment (runcom) [inline daemon/graphdriver/windows/windows.go]: does this miss anything after "to..."

File: daemon/graphdriver/windows/windows.go [hunk 3]
```diff
@@ -369,6 +380,7 @@ func (d *Driver) DiffSize(id, parent string) (size int64, err error) {
 	return archive.ChangesSize(layerFs, changes), nil
 }
 
+// RestoreCustomImages allows to add a custom image to the tag and graph store
```
Comment (swernli) [inline daemon/graphdriver/windows/windows.go]: Should be something like:
RestoreCustomImages adds any auto-detected OS specific images to the tag and graph store.

**Comment by vdemeester** [comment]
I wonder how I missed that on the last `golint` PR :sweat:.

It's probably nits but there is some comment that finish with a period, and some not :sweat_smile:.
Once those are fixed and the @swernli comment taken care of, LGTM for me too :wink: :blush:.

**Comment by calavera** [comment]
I've added punctuation marks all the way down, thanks :smile_cat:

File: daemon/graphdriver/windows/windows.go (hunk 2 above)
Comment (calavera) [inline daemon/graphdriver/windows/windows.go]: indeed

File: daemon/graphdriver/windows/windows.go (hunk 3 above)
Comment (calavera) [inline daemon/graphdriver/windows/windows.go]: fixed, thanks!

**Comment by swernli** [comment]
LGTM

**Comment by LK4D4** [comment]
LGTM

**Pull request merged**